docs/source/en/quicktour.md

```diff
@@ -10,314 +10,237 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
+from diffusers.quantizers import PipelineQuantizationConfig
 
+quant_config = PipelineQuantizationConfig(
+    quant_backend="bitsandbytes_4bit",
+    quant_kwargs={"load_in_4bit": True, "bnb_4bit_quant_type": "nf4", "bnb_4bit_compute_dtype": torch.bfloat16},
+    components_to_quantize=["transformer", "text_encoder_2"],
+)
+pipeline = DiffusionPipeline.from_pretrained(
+  "black-forest-labs/FLUX.1-dev",
+  torch_dtype=torch.bfloat16,
+  quantization_config=quant_config,
+).to("cuda")
+pipeline.enable_model_cpu_offload()
 
->>> def display_sample(sample, i):
-...     image_processed = sample.cpu().permute(0, 2, 3, 1)
-...     image_processed = (image_processed + 1.0) * 127.5
-...     image_processed = image_processed.numpy().astype(np.uint8)
-
-...     image_pil = PIL.Image.fromarray(image_processed[0])
-...     display(f"Image at step {i}")
-...     display(image_pil)
+prompt = """
+cinematic film still of a cat sipping a margarita in a pool in Palm Springs, California
+highly detailed, high budget hollywood movie, cinemascope, moody, epic, gorgeous, film grain
+"""
+pipeline(prompt).images[0]
+print(f"Max memory reserved: {torch.cuda.max_memory_allocated() / 1024**3:.2f} GB")
 ```
 
-To speed up the denoising process, move the input and model to a GPU:
+Refer to the [Reduce memory usage](./optimization/memory) docs to learn more about other memory reducing techniques.
 
-```py
->>> model.to("cuda")
->>> noisy_sample = noisy_sample.to("cuda")
-```
+### Inference speed
 
-Now create a denoising loop that predicts the residual of the less noisy sample, and computes the less noisy sample with the scheduler:
+The denoising loop performs a lot of computations and can be slow. Methods like [torch.compile](./optimization/fp16#torchcompile) increases inference speed by compiling the computations into an optimized kernel. Compilation is slow for the first generation but successive generations should be much faster.
 
-```py
->>> import tqdm
+The example below uses [regional compilation](./optimization/fp16#regional-compilation) to only compile small regions of a model. It reduces cold-start latency while also providing a runtime speed up.
 
->>> sample = noisy_sample
+Call [`~ModelMixin.compile_repeated_blocks`] on the model to activate it.
 
->>> for i, t in enumerate(tqdm.tqdm(scheduler.timesteps)):
-...     # 1. predict noise residual
-...     with torch.no_grad():
-...         residual = model(sample, t).sample
+```py
+import torch
+from diffusers import DiffusionPipeline
 
-...     # 2. compute less noisy image and set x_t -> x_t-1
-...     sample = scheduler.step(residual, t, sample).prev_sample
+pipeline = DiffusionPipeline.from_pretrained(
+  "black-forest-labs/FLUX.1-dev",
+  torch_dtype=torch.bfloat16
+).to("cuda")
 
-...     # 3. optionally look at image
-...     if (i + 1) % 50 == 0:
-...         display_sample(sample, i + 1)
+pipeline.transformer.compile_repeated_blocks(
+    fullgraph=True, dynamic=True
+)
```

Comment:
We should not be using `dynamic=True` unless the users are interested in doing inference with varying resolutions.